lib/list.gd

```diff
@@ -869,6 +869,44 @@ DeclareGlobalFunction( "PositionSet" );
 DeclareOperation( "PositionProperty", [ IsList, IsFunction ] );
 DeclareOperation( "PositionProperty", [ IsList, IsFunction, IS_INT ] );
 
+#############################################################################
+##
+#O  PositionMaximum( <list> [, <func>] )
+#O  PositionMinimum( <list> [, <func>] )
+##
+##  <#GAPDoc Label="PositionMaximum">
+##  <ManSection>
+##  <Oper Name="PositionMaximum" Arg='list [, func]'/>
+##  <Oper Name="PositionMinimum" Arg='list [, func]'/>
+##
+##  <Description>
+##  returns the position of maximum (with <Ref Func="PositionMaximum"/>) or
+##  minimum (with <Ref Func="PositionMinimum"/>) entry in the list <A>list</A>.
```

Comment:
It does not invalidate GAPDoc, but since both `PositionMaximum` and `PositionMinimum` are operations, it would be more consistent to use `Oper` instead of `Func` in `<Ref Func="PositionMaximum"/>` (there is even an automated check for such things added in #538).